### DISCUSSION TITLE
Should I assign GitHub Skills interactive courses via GitHub Classroom or Google Classroom?

### DISCUSSION THREAD

**Discussion by pcgallant** [community/community#50523] [GitHub Education]
Hi there - I want to have my students complete some of the great interactive interactive courses available at https://skills.github.com/ 

I was initially thinking I would assign them through GitHub classroom, but then realized that the GitHub Classroom will use the template to create a new repository with the student's name attached.

**However** the first step of the interactive course has a button to "Start course" which also has them create a new repository.  

I was thinking at first I could just tell them they can skip that step, but then realized that I think then the actions aren't triggered, so the course won't really start.  I haven't figured out how to use actions (yet) so am not sure if this is actually fixable.  Has anyone tried it this way that has a fork to share?

Since we also use Google Classroom, am I better off just attaching the course link to a Google classroom assignment and have them submit their link to the repository they created?  

BOTTOM LINE: I want to be able to see that they have actually completed each assigned course.
